### DISCUSSION TITLE
How To Run Bunjs in EC2 ?

### DISCUSSION THREAD

**Discussion by haseeb-ul-hassan-123** [community/community#136321] [Programming Help]
### Select Topic Area

General

### Body

How To Run Bunjs in EC2 -   i need to deploy with nginx & docker  it ASAP ??

**Comment by hi-ammad** [comment]
You Can Use Pm2
`pm2 start --interpreter ~/.bun/bin/bun index.ts`

**Comment by haseeb-ul-hassan-123** [comment]
For Docker Do i need to build containers in ec2 ?

**Comment by hi-ammad** [comment]
You Could Use Amazon ECS [https://aws.amazon.com/ecs/],) For Load Balancing You Can Go With Amazon-ECS Or Nginx.
